docs/source/tutorial_ph.md

```diff
@@ -0,0 +1,246 @@
+network-based modeling and problems involving multi-physics phenomena. We refer to {cite}`MU23`
+for a general introduction to port-Hamiltonian systems and their applications.
+
+We say a LTI system is *port-Hamiltonian* if it can be expressed as
+
+```{math}
+E \dot{x}(t) & = (J - R) Q x(t) + (G-P) u(t), \\
+y(t) & = (G+P)^T Q x(t) + (S-N) u(t),
+```
+
+where {math}`H := Q^T E`, and if
+
+```{math}
+\Gamma =
+\begin{bmatrix}
+    J & G \\
+    -G^T & N
+\end{bmatrix},
+\text{ and }
+W =
+\begin{bmatrix}
+    R & P \\
+    P^T & S
+\end{bmatrix}
+```
+
+satisfy
+{math}`H = H^T \succ 0`,
+{math}`\Gamma^T = -\Gamma`, and
+{math}`W = W^T \succcurlyeq 0`.
+
+The quadratic (energy) function {math}`\mathcal{H}(x) := \tfrac{1}{2} x^T H x`,
+typically called Hamiltonian, corresponds to the energy stored in the system. In
+applications, {math}`E` and/or {math}`Q` often are identity matrices.
+
+It is known that if the LTI system is minimal and stable, the following are equivalent:
+
+- The system is passive.
+- The system is port-Hamiltonian.
+- The system is positive real.
+
+See for example {cite}`BU22` for more details.
+
+In pyMOR, there exists a {{ PHLTIModel }} class. As of now, pyMOR only supports
+port-Hamiltonian systems with nonsingular E. {{ PHLTIModel }} inherits from
+{{ LTIModel }}, so {{ PHLTIModel }} can be used with all reductors which expect
+a {{ LTIModel }}. For model reduction, it is often desirable to preserve the
+port-Hamiltonian structure, i.e., to compute a ROM which is also port-Hamiltonian.
+
+A passive {{ LTIModel }} can be converted into a {{ PHLTIModel }} using
+the {func}`~pymor.models.iosys.PHLTIModel.from_passive_LTIModel` function if desired.
+Consequentely, one option to preserve port-Hamiltonian structure is to use a reductor
+which preserves passivity (but returns a ROM of type {{ LTIModel }}), and convert the
+ROM into a {{ PHLTIModel }} in a post-processing step.
+
+## A toy problem: Mass-spring-damper chain
+
+As a toy problem, we use a mass-spring-damper chain, which can be formulated
+as a port-Hamiltonian system (see {cite}`GPBV12`):
+
+```{image} msd_example.svg
+:alt: MSD example
+:width: 60%
+```
+
+Here, the spring constants are denoted by {math}`k_i` and the damping constants by
+{math}`c_i`, {math}`i=1,\dots,n/2`.
+The inputs {math}`u_1` and {math}`u_2` are the external forces on the first two
+masses {math}`m_1` and {math}`m_2`. The system outputs {math}`y_1` and {math}`y_2`
+correspond to the velocities of the first two masses {math}`m_1` and {math}`m_2`.
+The toy problem is included in pyMOR in the {mod}`pymor.models.examples` module as
+{func}`~pymor.models.examples.msd_example`.
+
+## Structure-preserving model order reduction
+
+pyMOR provides three reductors which can be used for model order reduction 
+while preserving the port-Hamiltonian structure:
+
+- pHIRKA ({class}`~pymor.reductors.ph.ph_irka.PHIRKAReductor`),
```

Comment:
```suggestion
- pH-IRKA ({class}`~pymor.reductors.ph.ph_irka.PHIRKAReductor`),
```

I think the "official" name (from the [paper](https://doi.org/10.1016/j.automatica.2012.05.052)) is IRKA-PH, but I guess PH-IRKA or pH-IRKA is also fine.